uber/templates/departments/form.html

```diff
@@ -275,6 +279,12 @@ <h1>
 <p>{{ department.description }}</p>
 {% if department.is_shiftless -%}
   <p>This department is <b>shiftless</b>; it does not track staff hours using shifts.</p>
+{%- else -%}
+{% if department.max_consecutive_hours == 0 -%}
+  <p>Members of this department may work <b>any number of hours</b> in a row</p>
```

Comment:
I think `<b></b>` is passe now, so this and line 286 should be `<strong></strong>`